rust/xaynet-sdk/src/settings/mod.rs

```diff
@@ -0,0 +1,26 @@
+use xaynet_core::{
+    crypto::SigningKeyPair,
+    mask::{MaskConfig, MaskConfigPair},
+};
+
+mod max_message_size;
+pub use max_message_size::MaxMessageSize;
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct PetSettings {
+    pub(crate) keys: SigningKeyPair,
+    pub(crate) mask_config: MaskConfigPair,
```

Comment:
No related to this PR:
I recently asked myself if it makes sense that the client receives the maskconfig from the coordinator instead of hardcoding it in the client. In this way, the coordinator user can adjust the config later without having to update the mobile app

Reply:
That would be nice indeed. Also I think this is the only part in the client that requires understanding of the PET protocol. If we get rid of that, all the complexity is on the server. @janpetschexain wdyt?

Reply:
that's a good idea, we can make them part of the round parameters. not sure why we didn't do this in the first place.